build.assets/tooling/cmd/please-run.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env sh
```

Comment:
Avoid shell injection:

```suggestion
#!/bin/sh
```